host-controller/src/main/java/org/jboss/as/host/controller/resources/StoppedServerResource.java

```diff
@@ -67,4 +94,19 @@ public void registerOperations(final ManagementResourceRegistration resourceRegi
         resourceRegistration.registerOperationHandler(CompositeOperationHandler.INTERNAL_DEFINITION, CompositeOperationHandler.INSTANCE);
     }
 
+    class LaunchTypeHandler implements OperationStepHandler {
+        // this is for operation in domain model and embed-host-controller mode
+        // org.jboss.as.server.operations.LaunchTypeHandler takes care of standalone mode and embed-server mode.
+        private final ServerEnvironment.LaunchType launchType;
```

Comment:
This field is not needed as it's always the same.

Reply:
Thanks, since this does not need to be parameterized launch type, I removed class LaunchTypeHandler, and put it into lambda as other OSH.